src/HAL/STM32F1/i2c.cpp

```diff
@@ -0,0 +1,38 @@
+#include "smuff.h"
```

Comment:
I'm pretty sure this will throw an error if compiled under Linux

Reply:
Fixed